elytron/src/main/resources/org/wildfly/extension/elytron/LocalDescriptions.properties

```diff
@@ -1340,6 +1340,16 @@ elytron.certificate-authority-account.credential-reference.store=The name of the
 elytron.certificate-authority-account.credential-reference.alias=The alias which denotes stored secret or credential in the store.
 elytron.certificate-authority-account.credential-reference.type=The type of credential this reference is denoting.
 elytron.certificate-authority-account.credential-reference.clear-text=Secret specified using clear text. Check credential store way of supplying credential/secrets to services.
+elytron.server-ssl-sni-context=A server side SNI Aware SSLContext that selects between an underlying context based on the provided SNI name
+elytron.server-ssl-sni-context.add=Adds a SNI context
+elytron.server-ssl-sni-context.remove=Removes a SNI context
+elytron.server-ssl-sni-context.default-ssl-context=The context to use if no SNI information is present, or if it does not match any mappings
+elytron.server-ssl-sni-context.sni-mapping=A mapping between a server name an an SSContext
+elytron.sni-mapping=A mapping between a server name an an SSContext
```

Comment:
the properties `elytron.sni-mapping` and `elytron.server-ssl-sni-context.sni-mapping` contain two typos each:
```
an -> and
SSContext -> SSLContext
```